velox/common/memory/MemoryArbitrator.h

```diff
@@ -425,6 +425,18 @@ class ScopedMemoryArbitrationContext {
   MemoryArbitrationContext currentArbitrationCtx_;
 };
 
+/// Object used to setup memory arbitration context for a leaf memory pool.
+class ScopedMemoryArbitration {
```

Comment:
Looking together with the above class, plus a scoped class containing another scoped class, it is quite confusing. Can we remove ScopedMemoryArbitrationContext class and put the impl of it inside of ScopedMemoryArbitration class?

Reply:
Chat offline. They are different.